VeriSign-C3SSA-G2-temporary-intermediate-after-1024bit-removal:2.16.47.0.110.205.23.112.102.231.95.163.130.10.121.31.5.174.crt

```diff
@@ -0,0 +1,33 @@
+# alias="VeriSign-C3SSA-G2-temporary-intermediate-after-1024bit-removal"
```

Comment:
Remove me :)

Reply:
Do we remove anything without an `openssl-trust` field at all?

Reply:
Rob says yes.